packages/react-components/react-tags/src/components/Tag/__snapshots__/Tag.test.tsx.snap

```diff
@@ -5,21 +5,13 @@ exports[`Tag renders a default state 1`] = `
   <div
     class="fui-Tag"
   >
-    <span
+    <div
```

Comment:
Can we remove the snapshot test in favor of a simple "it renders correctly" test that asserts the relevant parts of a simple, default render case? Or maybe just remove the test altogether in this case since the default `Tag` render is simple and conformance tests should cover the important parts like having the correct classes applied.

Examples:
- [Button](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-button/src/components/Button/Button.test.tsx#L36)
- [Field](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-field/src/components/Field/Field.test.tsx#L37)

Reply:
I removed the snapshot test altogether :) I think conformance tests are good enough for now